### DISCUSSION TITLE
Endpoint for Query Volume + Limits?

### DISCUSSION THREAD

**Discussion by tappandev** [openstates/issues#612] [Q&A] [answered]
Hey all - new to OpenStatesAPI, love the content!! I'm setting up some one-off project queries that I will run infrequently but will have decent volume when I do run (once every few months). I'm wondering if there is an endpoint to retrieve daily query volume for a given key (e.g., have run 100 of allotted 500 queries on 1/13/2022)? Thanks! Dave

**Comment by jamesturk** [comment, marked as answer]
We don't have an endpoint for this, but if you email ***@***.***
with your key & request we can sort out a higher limit for you.

**Reply by bdavisce21** [reply]
Hi James,
My company wants to pull the openstates.org federal and state legislator information into our database but we have some questions about the number of API calls and we would like to speak with someone regarding the setup and increasing the number of API calls.  Like Dave above, the content is amazing and exactly what we are looking for, we just need a little help.
Thanks,
Braden

**Comment by jamesturk** [comment]
Please email ***@***.*** to discuss this, going to close this
ticket out as it is better handled in private so people can share details.

**Reply by jamesturk** [reply]
contact at openstates dot org
